metricbeat/module/linux/ksm/data.go

```diff
@@ -0,0 +1,77 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package ksm
+
+import (
+	"path/filepath"
+
+	"github.com/elastic/beats/v7/metricbeat/module/linux"
+)
+
+type ksmData struct {
+	PagesShared      int64 `json:"pages_shared"`
+	PagesSharing     int64 `json:"pages_sharing"`
+	PagesUnshared    int64 `json:"pages_unshared"`
+	PagesVolatile    int64 `json:"pages_volatile"`
+	FullScans        int64 `json:"full_scans"`
+	StableNodeChains int64 `json:"stable_node_chains"`
+	StableNodeDups   int64 `json:"stable_node_dups"`
```

Comment:
why do we need `json` struct tags here?

Reply:
That struct is getting sent right to `report.Event` we need that for formatting, right?